### PR TITLE
revert port in GenericWebhook

### PR DESCRIPTION
**Pull request by MosheEichler**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/XSUP-36245)

## Description
Revert port to be required

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by thefrieddan1 — APPROVED** [review]

File: Packs/GenericWebhook/ReleaseNotes/1_0_31.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Generic Webhook
+
+Updated the **Listen Port** parameter to be required.
```
Comment (julieschwartz18) [inline Packs/GenericWebhook/ReleaseNotes/1_0_31.md:6]: recommend italics instead of bold (the convention for a parameter name):
*Listen Port*

**Comment by julieschwartz18** [comment]
TW review complete. One minor comment.

**Comment by MosheEichler** [comment]
@ShahafBenYakir needs a force merge because it fails on BC
Screenshot 2024-04-30 at 19 38 07

**Review by DeanArbel — APPROVED** [review]

File: Packs/GenericWebhook/Integrations/GenericWebhook/GenericWebhook.yml [hunk 2]
```diff
@@ -14,11 +14,11 @@ configuration:
...
   required: false
-- additionalinfo: "Runs the service on this port from within Cortex XSOAR. Requires a unique port for each long-running integration instance. Do not use the same port for multiple instances. Note: If you click the test button more than once, a failure may occur mistakenly indicating that the port is already in use. (For Cortex XSOAR 8 and Cortex XSIAM) If you do not enter a Listen Port, an unused port for Generic Webhook will automatically be generated when the instance is saved. However, if using an engine, you must enter a Listen Port."
+- additionalinfo: "Runs the service on this port from within Cortex XSOAR. Requires a unique port for each long-running integration instance. Do not use the same port for multiple instances. Note: If you click the test button more than once, a failure may occur mistakenly indicating that the port is already in use."
   display: Listen Port
   name: longRunningPort
   type: 0
-  required: false
+  required: true
   section: Connect
 - additionalinfo: "The username and password is required in XSOAR 8.5 and above"
```
Comment (JudahSchwartz) [inline Packs/GenericWebhook/Integrations/GenericWebhook/GenericWebhook.yml:23]:
```suggestion
- additionalinfo: "The username and password is required in XSOAR 8.5 and above. Using the `_header:` feature without using an engine will not work."
```

**Pull request merged**